docs/source-app/workflows/byoc/index.rst

```diff
@@ -5,62 +5,270 @@ Run Apps on your own cloud (BYOC)
+cloud/cluster will show you that the Lightning cluster works.
+
+At a high level, to accomplish that, you need to do these things:
+
+#. Give Lightning permission to act on your behalf in your AWS account.
+#. Install Lightning.
+#. Create a Lightning cluster and then run an App on your cluster.
+
+----
+
+*************
+Prerequisites
+*************
+
+For Lightning and Lightning BYOC:
+
+* Python 3.8.x or later (3.8.x, 3.9.x, 3.10.x) for Lightning installation
+
+* An active AWS account with enough permissions to create AWS IAM roles
+
+(Optional) For Terraform only:
+
+* Terraform installed
+
+* AWS CLI installed
+
+* AWS credentials to sign in to AWS
+
+----
+
+***********************
+Step 1: Set up AWS Role
+***********************
+
+**Who should do this:** IT department or anyone with admin access to the cloud provider.
+
+The first step is to create a role with permissions that allow Lightning to make changes to your AWS resources on your behalf.
+
+You can do that in two ways:
+
+* Directly in AWS
+
+* Terraform (recommended for Terraform users)
+
+----
+
+Method 1: Use the AWS Console to create role
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You're going to create a custom role, that uses a custom policy, in AWS using the AWS management console.
+
+These are the settings you'll specify when creating the role:
+
+* **Trusted entity type:** **AWS account**
+
+* **Another AWS account number:** **748115360335**
+
+    .. note::
+
+        This is Lightning AI's account number. It allows Lightning AI to make changes to your AWS resources
+		(like creating EC2 instances for your BYOC cluster) on your behalf.
+
+* **Require external ID:** Provide a secure external ID.
+
+    .. warning::
+
+        AWS does not treat the external ID as a secret. But Lightning AI uses the external ID to access AWS on your behalf. **Do not share the external ID.** To read more on what the AWS external ID is and why it's useful go `here <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html>`_.
+
+* Copy-paste the `contents of this JSON file <https://github.com/Lightning-AI/terraform-aws-lightning-byoc/blob/main/iam_policy.json>`_ into the JSON workspace
+  for the custom policy that the custom role uses.
+
+   .. note:: Ignore the warnings generated by AWS when you create the policy.
+
+If you are an IT person creating this role for someone else (researchers, data scientists, Lightning App creators), assign the role to their AWS account after creating the role.
+
+Here's a short video showing you the process.
+
+.. raw:: html
+
+   <video id="background-video" autoplay loop muted controls poster="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/aws_signin.png" width="100%">
+      <source src="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/byoc_aws_setup_shorter.mp4" type="video/mp4" width="100%">
```

Comment:
Is this the right long-term storage place for the video? pl-flash-data?

Reply:
@nmiculinic This was what I was provided to use. There are actually a lot of videos and images stored in this S3 bucket.

Reply:
CC: @awaelchli where should the docs data be stored?

Reply:
We use the pl-flash-data bucket for public resources that are too heavy to go into version control, like images, video etc. The name is just an artifact of the past.